src/App.js

```diff
@@ -4,7 +4,7 @@ import './App.css';
 function App() {
```

Comment:
You should use `jsx` files for your components and not `js`.

Reply:
Okay sir, noted